src/index.ts

```diff
@@ -225,44 +142,35 @@ async function main() {
   }
 
   try {
-    if (conf.useMercury && redis) {
-      const checkNetwork = "PUBLIC";
-      // initial token and userID not needed for integrity checks
-      const integrityCheckMercurySession = {
-        renewClientMaker,
-        backendClientMaker,
-        currentDataClientMaker,
-        backends,
-        credentials: {
-          TESTNET: {
-            email: conf.mercuryEmailTestnet,
-            password: conf.mercuryPasswordTestnet,
-          },
-          // need to set this to the integrity check accounts for Mercury to remove entries periodically
-          PUBLIC: {
-            email: conf.mercuryIntegrityCheckEmail,
-            password: conf.mercuryIntegrityCheckPass,
-          },
+    // the worker is not properly instantiated when running this app with ts-node
```

Comment:
I haven't found a great way to get this to work with out dev server, I think for now the easiest thing to do is to build and run the app manually using the webpack build when we need to test/work on the integrity check. It's a little cumbersome but not sure how to make it easier yet. Thoughts? @piyalbasu

Reply:
Interesting - yeah, I don't have a great solution off the top of my head. Since we'll likely be making more changes to the endpoints rather than the integrity checks, maybe this friction is fine for now. If it becomes a problem, we could always come back to this